Gemfile.lock

```diff
@@ -45,6 +45,11 @@ PATH
     samson_kubernetes (0.0.1)
 
 PATH
+  remote: plugins/ledger
+  specs:
+    ledger (0.0.1)
```

Comment:
gem itself should be called samson_ledger to not collide with https://rubygems.org/gems/ledger